src/features/wallet/send/SendView.tsx

```diff
@@ -420,33 +372,43 @@ const SendView = ({ scanResult, sendType, hotspot, isSeller }: Props) => {
     }
   }
 
-  const constructTxn = async () => {
+  const constructTxns = async () => {
     if (type === 'payment') {
-      return makePaymentTxn(balanceAmount.integerBalance, address, getNonce())
+      return Promise.all(
```

Comment:
So your goal of allowing multiple address / payee fields in a QR for payment was to be able to create a QR that can send multiple addresses different amounts correct?

The logic here is not the correct way to go about that. For a payment with multiple payees and amounts, we need to pass them all into one payment_v2 transaction. This is something we currently support on the blockchain but haven't done in the app yet. 

Instead of creating multiple transaction like you have done, we need to create one transaction and fill the payee and amounts correctly in the payments array of that single payment_v2 transaction.

You will need to revert this change and make updates to https://github.com/helium/hotspot-app/blob/main/src/utils/transactions.ts#L26 to allow multiple payee's and amounts.

Reply:
The changes you made look good!